install_bootstrap

```diff
@@ -218,7 +221,7 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
       dcc_dashboard_port=8080
 
       if [ -f boardwalk_launcher_config/boardwalk.config ] ; then
-	  source <(jq -r "to_entries|map(\"\(.key)=\(.value|tostring)\")|.[]" boardwalk_launcher_config/boardwalk.config)
+          source <(jq -r "to_entries|map(\"\(.key)=\(.value|tostring)\")|.[]" boardwalk_launcher_config/boardwalk.config)
```

Comment:
You should not change this line unless you mean to. I think your editor is converting tabs to spaces or something like that. Please revert this change.

Reply:
I changed it back. Although I believe that the entire file has space except line 221.

Reply:
I mean, my aim was only to make it consistent...

Reply:
It's not changed back, but I suppose you can leave it.

While it's the right change to make, combining it with unrelated changes in the same commit is confusing, because looking back at this will assume this was change was necessary to get Boardwalk running in dev mode.

But it's small enough to just leave. 

But as a general practice, you shouldn't include unrelated formatting changes; or if you do, at least call them out in the commit message.

Reply:
Okay, I'll follow this going forward. Yes, I saw your comment, and slacked you a histogram of all the length of all commit messages in that repo. That looks very different from the one in the link.

Reply:
It doesn't look like we're following it. I'm on a one person crusade to change it. :)

I guess there is no SOP on this though, so I probably went overboard, and you can skip.

But I do think your messages should be informative, even if they don't follow the format. and it is informative now.